wfe2/wfe.go

```diff
@@ -1647,7 +1643,7 @@ func (wfe *WebFrontEndImpl) Certificate(ctx context.Context, logEvent *web.Reque
 	logEvent.Extra["RequestedSerial"] = serial
 	beeline.AddFieldToTrace(ctx, "request.serial", serial)
 
-	cert, err := wfe.SA.GetCertificate(ctx, serial)
+	cert, err := wfe.SA.GetCertificate(ctx, &sapb.Serial{Serial: serial})
```

Comment:
I noticed we are not checking to see if the returned cert object is nil (even if the error is nil) as the wrapper client did, perhaps in the event of a GRPC error or SA regression.

Reply:
gRPC actually guarantees that, if err is nil, then the returned message is non-nil, so we don't need to check the top-level object. It could be an empty message, but in this case the only piece of the cert that we actually care about is the `cert.Der`, and that'll throw an appropriate error on line 1680 if that's a zero-value.

Reply:
Ok, good enough for me. Thanks!